src/views/pengajuanLogistik/detail.vue

```diff
@@ -303,6 +324,22 @@ export default {
     const temp = this.detailLogisticRequest.letter.letter.split('.')
     this.letterFileType = temp[temp.length - 1]
     this.isVerified = this.detailLogisticRequest.applicant.verification_status === 'Terverifikasi'
+    this.listLogisticNeeds.forEach(element => {
+      if (element.status === 'approved') {
+        element.status = this.$t('label.approved')
+      } else if (element.status === 'not_delivered') {
+        element.status = this.$t('label.not_delivered')
+      } else if (element.status === 'delivered') {
+        element.status = this.$t('label.delivered')
+      } else if (element.status === 'not_available') {
+        element.status = this.$t('label.not_available')
+      } else {
+        element.status = this.$t('label.not_available')
+      }
+    })
```

Comment:
coba ganti pake switch code ini, agar mudah dibaca, lalu test dulu:

```suggestion
    this.listLogisticNeeds.forEach(element => {
      switch (element.status) {
        case 'approved':
          element.status = this.$t('label.approved')
          break;
        case 'not_delivered':
          element.status = this.$t('label.not_delivered')
          break;
        case 'delivered':
          element.status = this.$t('label.delivered')
          break;
        case 'not_available':
          element.status = this.$t('label.not_available')
          break;
        default:
          element.status = this.$t('label.not_available')
      }
    })
```